src/SDK.ts

```diff
@@ -321,7 +321,7 @@ export async function getService<T>(contributionId: string): Promise<T> {
 * @param instance - Either: (1) an object instance, or (2) a function that takes optional context data and returns an object instance.
 */
 export function register<T = any>(instanceId: string, instance: T): void {
-    parentChannel.getObjectRegistry().register(instanceId, instance);
+    parentChannel.getObjectRegistry().register(instanceId, instance as Object);

```

Comment:
```suggestion
    parentChannel.getObjectRegistry().register(instanceId, instance);

```
with the above suggestion this cast is unnecessary.